externs/olx.js

```diff
@@ -303,6 +303,45 @@ olx.MapOptions.prototype.view;
 
 
 /**
+ * Object literal with options for the forEachFeatureAtPixel and
+ * hasFeatureAtPixel methods.
```

Comment:
You could use: `{@link ol.Map#forEachFeatureAtPixel}` and `{@link ol.Map#hasFeatureAtPixel}`